### PR TITLE
[GLUTEN-7359][VL] Enable partial project in RAS

### PR DESCRIPTION
**Pull request by zhztheplayer**
Following #7360, enable the feature in RAS.

### CODE REVIEW

**Comment by zhztheplayer** [comment]
cc @jinchengchenghh

File: backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala [hunk 1]
```diff
@@ -157,7 +157,7 @@ case class ColumnarPartialProjectExec(original: ProjectExec, child: SparkPlan)(
       return ValidationResult.failed("Contains expression not supported")
     }
     if (
-      ExpressionUtils.isComplexExpression(
+      ExpressionUtils.hasComplextExpressions(
```
Comment (jinchengchenghh) [inline backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala:160]: a typo hasComplextExpressions -> hasComplexExpressions

Comment (zhztheplayer) [reply backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala:160]: Thanks for catching.

**Review by jinchengchenghh — APPROVED** [review]
Thanks!

**Pull request merged**